extensions/GlobalMessages/tests/GlobalMessages0.data.php

```diff
@@ -0,0 +1,970 @@
+
+'''What are Kudos?'''
+: If you find a particular discussion or reply interesting, well thought out, or amusing you can show direct appreciation by giving it Kudos. They can be helpful in voting situations, too.
+
+'''What are Topics?'''
+: Topics allow you to link a forum discussion with a wiki article. It's another way to keep Forums organized and to help people find interesting discussions. For example, a Forum thread tagged with ''Lord Voldemort'' will appear at the bottom of the ''Lord Voldemort'' article.",
+// 143 -------------------------------------------------------------------------
+'founderprogressbar-wikialabs-visit-description' => 'Labs allows you to turn on the newest features from Fandom on your wiki. These features are all in beta, but it\'s a good way to see what\'s coming up from Fandom and try it out yourself.',
+// 144 -------------------------------------------------------------------------
+'founderprogressbar-wikialabs-visit-label' => 'Visit Labs',
+// 145 -------------------------------------------------------------------------
+'global-navigation-create-wiki' => 'Start a community',
+// 146 -------------------------------------------------------------------------
+'global-navigation-explore-wikia-link-label' => 'Explore Fandom',
+// 147 -------------------------------------------------------------------------
+'global-navigation-global-search' => 'Search all of Fandom',
+// 148 -------------------------------------------------------------------------
+'global-navigation-local-search' => 'Search this community',
+// 149 -------------------------------------------------------------------------
+'global-shortcuts-caption-start-a-new-wikia' => 'Start a new wiki',
+// 150 -------------------------------------------------------------------------
+'global-shortcuts-category-current-wikia' => 'Current community',
+// 151 -------------------------------------------------------------------------
+'group-helper' => 'Fandom Helpers',
+// 152 -------------------------------------------------------------------------
+'group-helper-member' => 'Fandom Helper',
+// 153 -------------------------------------------------------------------------
+'group-staff' => 'Fandom Staff',
+// 154 -------------------------------------------------------------------------
+'group-staff-member' => 'Fandom Staff',
+// 155 -------------------------------------------------------------------------
+'group-translator' => 'Fandom Translators',
+// 156 -------------------------------------------------------------------------
+'group-translator-member' => 'Fandom Translator',
+// 157 -------------------------------------------------------------------------
+'group-util' => 'Fandom Utilities',
+// 158 -------------------------------------------------------------------------
+'group-util-member' => 'Fandom Utilities',
+// 159 -------------------------------------------------------------------------
+'hub-Wikia' => 'Fandom',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Fandom" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)